com.unity.mobile.notifications/Runtime/iOS/iOSNotification.cs

```diff
@@ -181,8 +179,13 @@ public int Badge
         /// </summary>
         public string Data
         {
-            get { return data.data; }
-            set { data.data = value; }
+            get { return userInfo["data"]; }
+            set { userInfo["data"] = value; }
+        }
+
+        public Dictionary<string, string> UserInfo
```

Comment:
Is user allowed to change this data? if not - consider replacing with IReadOnlyDictionary<string, string>

Reply:
It is allowed to modify the dictionary. Added API doc in latest commit.